lib/semian/net_http.rb

```diff
@@ -36,6 +36,7 @@ def semian_identifier
       ::Timeout::Error, # includes ::Net::ReadTimeout and ::Net::OpenTimeout
       ::SocketError,
       ::Net::HTTPBadResponse,
+      ::Net::HTTPFatalError,
```

Comment:
If you include it here by default, why does your option matter? I think your option should ensure that `::Net::HTTPFatalError` is in resource exceptions. My impression of reading this code would be that it _will_ open the circuit, but it won't raise..?